lib/cpp/antaris_sdk_environment.cc

```diff
@@ -107,9 +106,16 @@ static void parse_a_conf(char *conf_line, conf_t *out_conf)
 
 static void update_a_conf(char *conf_line)
 {
+    int8_t length = -1;
     conf_t a_conf = {0};
+    size_t length = 0;
 
     parse_a_conf(conf_line, &a_conf);
+    length = strnlen(a_conf.value, MAX_IP_OR_PORT_LENGTH);
+    if(length == MAX_IP_OR_PORT_LENGTH) {
+        printf("Property value is greater than %d characters is not supported. Going with default value \n", MAX_IP_OR_PORT_LENGTH);
```

Comment:
Can we ASSERT here in #ifdef DEBUG mode because we never expect such error in config file.